README.md

```diff
@@ -71,6 +71,47 @@ foo@bar:~$ npm start
 foo@bar:~$ npm run clean
 ```
 
+## Running data_acquisition.py
+If you don't have a local instance of the PostgreSQL table yet, navigate to the `server` directory and run
+```console
+foo@bar:~$ npm start
+```
+to create an empty `job` table.
+
+In a different tab, navigate to the `data_acquisition` directory.
+
+Use [pipenv](https://pipenv.readthedocs.io/en/latest/), a Python dependency manager, to install any missing requirements. Note that we're using Python 3.
+```console
+foo@bar:~$ pip install pipenv
+foo@bar:~$ pipenv install
+```
+
+You'll have to add the ZipRecruiter API key as an environment variable. This isn't the **most** secure way of storing it, but better than putting it directly in the code.
+
+To get access to the shell for the managed environment:
+```console
+foo@bar:~$ pipenv shell
+```
+
+Then open its `.env` file:
+```console
+(data_acquisition) foo@bar:~$ vim .env
```

Comment:
Instead of opening `.env` in a text editor, I would recommend doing something like:
```bash
echo "export ZIPRECRUITER_API_KEY=\"blahblah\"" >> .env
```
which will have the same effect but should be a bit more foolproof (e.g. for people that don't know how to use Vim).

Reply:
Also, it might be worth considering a small shell script like `install.sh` to do all this setup for people; that could look like
```
#!/usr/bin/env bash
set -euxo pipe fail
pip3 install pipenv
pipenv install --dev
echo "export ZIPRECRUITER_API_KEY=\"blahblah\"" >> .env
```
which could then be run with
```bash
$ chmod +x install.sh
$ ./install.sh
```
though that might also break in unexpected ways.

Reply:
By the way, is there a reason you're starting `pipenv shell` to create `.env`? I'm pretty sure the `.env` file should work just fine as long as it exists in the directory you run `pipenv` commands in (i.e. even if you create it from outside the virtualenv).

Reply:
I ran into issues where the script was unable to find the variable sometimes and it was happening randomly. Doing it in `pipenv shell` seemed to fix it.

Reply:
Oh, you know what the problem might be? That's not the correct format for `.env` files - they should be like:
```bash
ZIPRECRUITER_API_KEY=blahblahblah
```
so probably Pipenv (when running with `pipenv run`) went "welp, no environment variables here," but when running inside `pipenv shell`, it accidentally ran the `export` command while parsing the `.env` file, thereby importing that environment variable into the virtualenv.

So this line should be corrected to, say:
```bash
echo "ZIPRECRUITER_API_KEY=blahblahblah" >> .env
```

Reply:
I'm not sure what that adds? Pipenv reads from `.env` files already.

Reply:
I'm not sure what that adds? Pipenv reads from `.env` files already.

Reply:
oh it just seems like youre encountering inconsistencies with `pipenv`

Reply:
we spent a while fiddling with random ways of setting env variables until tom just set up an env lib and we never thought about it again

Reply:
Yeah, I'm _pretty_ sure it's the `export` thing - in face I suspect `pipenv` uses `python-dotenv` behind the scenes anyway 😛